Civi/Payment/PropertyBag.php

```diff
@@ -65,6 +65,7 @@ class PropertyBag implements \ArrayAccess {
     'frequency_interval'          => 'recurFrequencyInterval',
     'recurFrequencyUnit'          => TRUE,
     'frequency_unit'              => 'recurFrequencyUnit',
+    'subscriptionId'              => 'recurProcessorID',
```

Comment:
Great :-) Nice and simple

Reply:
Yep the magic is in how Rich implemented it

```
  /**
   * Implements ArrayAccess::offsetGet
   *
   * @param mixed $offset
   * @return mixed
   */
  public function offsetGet ($offset) {
    $prop = $this->handleLegacyPropNames($offset);
    return $this->get($prop, 'default');
  }
```